features/support/worlds/broker_agency_world.rb

```diff
@@ -21,6 +21,26 @@ def assign_broker_agency_account
     employer_profile.benefit_sponsorships << broker_agency_account
     employer_profile.organization.save!
   end
+
+  def broker(*traits)
+    attributes = traits.extract_options!
+    @broker_organization ||= FactoryGirl.create(
+      :benefit_sponsors_organizations_general_organization,
+      :with_broker_agency_profile,
+      attributes.merge(site: site)
+    )
+  end
+
 end
 
 World(BrokerAgencyWorld)
+
+Given(/^there is a Broker (.*?)$/) do |legal_name|
+  broker legal_name: legal_name,
+         dba: legal_name
+  #benefit_sponsorship(broker)
+end
+
+And(/^the person is assigned to a broker agency$/) do
```

Comment:
Can we change person to broker just so its more clear to new developers whats happening?

Reply:
Yes that makes sense. I'll change that.